tests/unit/loader/test_tf_dataloader.py

```diff
@@ -64,15 +62,16 @@ def test_nested_list():
 
     batch = next(iter(train_dataset))
     # [[1,2,3],[3,1],[...],[]]
-    nested_data_col = tf.RaggedTensor.from_row_lengths(
-        batch[0]["data"][0][:, 0], tf.cast(batch[0]["data"][1][:, 0], tf.int32)
+    nested_data_col = tf.RaggedTensor.from_row_splits(
+        batch[0]["data__values"], tf.cast(batch[0]["data__offsets"], tf.int32)
```

Comment:
Using TensorTable here would remove the need to specify the particular naming convention we have adopted for the dictionary keys containing values/offsets. 

```python
X = TensorTable(batch[0])

tf.RaggedTensor.from_row_splits((X["data"].values, X["data"].offsets, tf.int32)
```

Reply:
Is it fine to leave the test as is ? I was not familiar with TensorTable